### PR TITLE
Restyled the UI

### PR DESCRIPTION
**Pull request by Hugovs2000**
Screenshot 2024-07-18 at 16 34 49
Screenshot 2024-07-18 at 16 35 01
Screenshot 2024-07-18 at 16 35 13
Screenshot 2024-07-18 at 16 35 37
Screenshot 2024-07-19 at 10 11 01
Screenshot 2024-07-18 at 16 35 51

### CODE REVIEW

File: src/app/components/itinerary-form/itinerary-form.component.html [hunk 1]
```diff
@@ -82,14 +83,14 @@ <h3 class="text-lg font-bold">Create a Tag!</h3>
       <div class="mb-4 flex flex-col">
         <button
           type="button"
-          class="btn mb-2 w-fit self-center bg-white"
+          class="mb-2 w-fit self-center rounded-t-lg border-b-[1px] border-white bg-slate-1000 p-4 text-white hover:bg-transparent"
```
Comment (yanevdw) [inline src/app/components/itinerary-form/itinerary-form.component.html]: You could just use ```border-b``` to set the bottom border to 1px.

**Review by yanevdw — APPROVED** [review]
Looks really good. Well done!

File: src/app/components/itinerary-form/itinerary-form.component.html (hunk 1 above)
Comment (Hugovs2000) [reply src/app/components/itinerary-form/itinerary-form.component.html]: Thank you 🙏🏻

**Review by AyaazStreet — APPROVED** [review]
Looks really nice 🤌🏻. You could maybe adjust the landing page heading so that "the" is not on a line on its own when on mobile.

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,42 +1,112 @@
...
-          class="btn btn-ghost h-8 min-h-8 text-sm text-blue-800"
-          aria-label="Log in button"
-          (click)="signOut()">
-          Log Out
-        </button>
-      } @else {
+<div class="drawer">
+  <input id="my-drawer" type="checkbox" class="drawer-toggle" />
+  <div class="drawer-content">
+    <p>
```
Comment (theadley) [inline src/app/app.component.html]: Why's this in a paragraph?

**Review by LesediMoko — APPROVED** [review]
Amazing work bro! Super impressed 👏

File: src/app/components/home/home.component.html [hunk 3]
```diff
@@ -37,38 +47,62 @@
...
-          <h3 class="text-lg font-bold">Hold on!</h3>
-          <p class="py-4">Are you sure you would like to delete this trip?</p>
+          <h3 class="text-2xl font-bold">Hold on!</h3>
+          <div class="py-4">
+            <p class="text-lg">
+              Are you sure you would like to delete
+              {{ selectedTrip()?.title ?? 'this trip' }}?
+            </p>
+            <p class="pt-2">
+              You will be deleting the associated itineraries as well.
```
Comment (theadley) [inline src/app/components/home/home.component.html]: Wording is funky. The itinerary is 1 to 1 with a trip, so you can't say "itineraries" you can say "activities" rather. Or just use the singular "itinerary"

File: src/app/components/trip-details/trip-details.component.html [hunk 4]
```diff
@@ -15,25 +45,58 @@ <h1 class="text-2xl font-bold">{{ trip.title }}</h1>
...
+                }
+              } @else {
+                @if (date | hasActivity: trip.itinerary ?? []) {
+                  <li class="tripDate rounded-full bg-green-1000 text-black">
+                    {{ date.getDate() }}
+                  </li>
+                } @else {
+                  <li class="tripDate">
+                    {{ date.getDate() }}
+                  </li>
```
Comment (theadley) [inline src/app/components/trip-details/trip-details.component.html:99]: OR:
```
<li [class]="'tripDate' + (date | hasActivity: trip.itinerary ?? []) ? 'rounded-full bg-green-1000 text-black' : ''">
  {{ date.getDate() }}
</li>
```

Comment (theadley) [reply src/app/components/trip-details/trip-details.component.html:99]: This is not an instruction but an alternative

Comment (theadley) [reply src/app/components/trip-details/trip-details.component.html:99]: Another idea is:
`[class.rounded-full]="date | hasActivity: trip.itinerary ?? []"`
`[class.bg-green-1000]="date | hasActivity: trip.itinerary ?? []"`
etc

File: src/app/components/trip-details/trip-details.component.html [hunk 5]
```diff
@@ -85,28 +153,32 @@ <h2 class="mb-2 text-xl font-bold">Itineraries:</h2>
...
-      <a type="button" class="btn btn-success" routerLink="/home">Done</a>
     </div>
     <dialog #confirmDeleteActivityModal class="modal">
       <div class="modal-box">
-        <h3 class="text-lg font-bold">Hold on!</h3>
-        <p class="py-4">Are you sure you would like to delete this activity?</p>
+        <h3 class="text-2xl font-bold">Hold on!</h3>
+        <p class="py-4">
+          Are you sure you would like to delete
+          {{ activityToDelete()?.title }} from {{ tripToUpdate()?.title }}?
```
Comment (theadley) [inline src/app/components/trip-details/trip-details.component.html]: This is the only one with no fallbacks (nullish coalescing)

**Review by theadley — CHANGES_REQUESTED** [review]
Fix it and then get the refactor tractor started

**Review by theadley — APPROVED** [review]

**Pull request merged**